kplclient.go

```diff
@@ -19,15 +24,25 @@ func NewKPLClient(host, port string) *KPLClient {
 
 //KPLClient represents a client to the KPL Server
 type KPLClient struct {
-	Host    string
-	Port    string
-	Started bool
+	Host string
+	Port string
+	// ErrPort is the optional field, which is provided, will cause a server to start and
+	// on this port and retrieve any error.
+	// Provide ErrHandler if ErrPort is set.
+	ErrPort    string
+	ErrHandler func(data string)
+	Started    bool
 }
 
 //Start starts up a communication channel to the server
 func (c *KPLClient) Start() error {
 
 	if !c.Started {
+		if c.ErrPort != "" {
+			go c.processErrMessage()
+			time.Sleep(time.Second * 30)
```

Comment:
Why do you sleep here?

Reply:
Ah! This is a mistake. Will remove this. Was added to allow testing in local.